lib/vyasa/medium/track.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Vyasa.Medium.Track do
```

Comment:
* intention for tracks is to be a personal playlist
* can be used to store a playlist for a source / chapter as well 

querying for voices doesn't need to rely on tracks

a user playlist is a track  

TODO: 
hide the previous and after buttons on the player temporarily, until we're ready with multiple